src/components/edit/earthEngine/StyleSelect.js

```diff
@@ -11,40 +11,48 @@ import { setParams } from '../../../actions/layerEdit';
 import { EE_BUFFER } from '../../../constants/layers';
 import styles from '../styles/LayerDialog.module.css';
 
-const getStepsFromParams = ({ palette, min }) =>
-    palette.split(',').length - (min === 0 ? 1 : 2);
+const minSteps = 3;
+const maxSteps = 9;
 
-const isValidSteps = (newSteps, params) => {
-    const steps =
-        newSteps !== undefined ? newSteps : getStepsFromParams(params);
-    return steps > 0 && steps < 8; // Valid steps: 1-7
-};
+const isValidParams = ({ min, max }) =>
```

Comment:
`params` are plural so should be `areValidParams`? or `paramsAreValid`? Thoughts from a native speaker like @jenniferarnesen ?

Reply:
Good question! I don't think being a native speaker makes me more of an authority on this though (but please do ask me about when to use which vs that if you want a pedantic answer!)

I don't have a strong opinion on it - maybe it feels more natural to say "paramsAreValid" when reading the code aloud to yourself.

Reply:
Changed to "paramsAreValid"